storage/innobase/log/log0recv.cc

```diff
@@ -1781,14 +1787,25 @@ static byte *recv_parse_or_apply_log_rec_body(
           /** else the index is flushed to disk before
           backup started hence no error */
         } else {
+          // While scaning redo logs during a backup operation a
+          // MLOG_INDEX_LOAD type redo log record indicates, that a DDL
+          // (create index, alter table...) is performed with
+          // 'algorithm=inplace'. If using pagetracking, the affected tablespace
+          // must be copied using full scan. Record it in the index_load_list
+          // We do this during the first scan of redo before starting file copy
+          // thread
```

Comment:
from the pages copied by page-tracking list. Since in place DDL skips redo logging and the pages that are not yet flushed, they will not be tracked by page-tracking. And since the redo logging is skipped, PXB will fail to get those changed pages (currently only in buffer pool).

Hence, we rely on re-copying the datafiles.

Reply:
added the comment.